.env.base

```diff
@@ -152,5 +152,10 @@ REACT_APP_SNAP_ID=npm:@shapeshiftoss/metamask-snaps
 
 REACT_APP_EXPERIMENTAL_MM_SNAPPY_FINGERS=true
 
+REACT_APP_SNAP_ID=npm:@shapeshiftoss/metamask-snaps
```

Comment:
TODO: what's up with this diff?

Reply:
Interactively dropped the additional commits that were present here